src/lib/card/card.scss

```diff
@@ -18,7 +18,7 @@ md-card {
 }
 
 md-card:hover {
-  @include md-elevation(2);
+  @include md-elevation(8);
```

Comment:
cards should not change elevation when hovered over. The [spec](https://material.google.com/material-design/elevation-shadows.html#elevation-shadows-elevation-android) language there says a _picked up_ card should have an elevation of 8. This is a special type of card that is meant to be interacted with, not a general-purpose card.